.env

```diff
@@ -0,0 +1,2 @@
+DATABASE_CONNECTION="mongodb+srv://AMesser:bGHHP9MNANRkKS3a@cluster0.hrluy6c.mongodb.net/"
```

Comment:
You don't want to check this .env file into source control either, this is still exposing your DB credentials. You should add `.env` to your `.gitignore` file so this never gets committed.

Reply:
.env added to git ignore file